### PR TITLE
Rename almost every instance of Kassy to Concierge

### LINKED ISSUE #100: New name for Kassy and organisation migration
The name `Kassy` no longer really represents what this project is about so we should get a new one. 
We also need to migrate to an organization so that we can split pre-installed modules into their own repositories.

### LINKED ISSUE #174: Localisation
What about some functionality to add localisations and translations? For Kassy and modules, with ability to set language in config (missing strings would be taken from English and there could be placeholder string specified in code in case it's missing in all lang files) There could be lang folder for translations and they could also be put into modules' folders for custom modules. That could be good feature, as Kassy's getting bigger. I could personally translate Kassy to Polish.
I don't know how I can mark this issue as feature or enhancement, sorry, I'm pretty new to Git.

### LINKED ISSUE #189: Make help optional
As much as help is something which should be encouraged, there are genuine use cases for not providing it.

### LINKED ISSUE #191: event_source incorrectly reports Skype as Facebook
@mrkno No idea where/why this is happening on 4.0alpha:

```
{ thread_id: '8:phillipadsmith',
  sender_id: 'phillipadsmith',
  sender_name: 'Phillip Smith',
  body: '!start',
  event_source: 'facebook',
  arguments: [ '!start' ],
  arguments_body: '',
  module_match_count: 1 }
```

That was sent from Skype. 

This only happens if I'm running several integrations together, e.g., Telegram, Facebook, and Skype. 

Basically, it looks like event.source gets set once on startup in integrations.js and then doesn't get updated on subsequent events.

Is there a quick fix? I've poked at it and can't see anything obviously wrong. :-/

### PR DESCRIPTION
**Pull request by mrkno**
- Does a full rename of `Kassy` to `Concierge` (including repository). Resolves #100.
- Fix `event_source`. Resolves #191.
- Makes help optional. Resolves #189.
- Makes `kassy.json` help translatable and `pl` translations UTF-8. Resolves #174.

### CODE REVIEW

**Comment by phillipadsmith** [comment]
Looking good. :-)

**Comment by mrkno** [comment]
@Awarua- this is reviewable and mergable now

**Comment by Awarua-** [comment]
Approved

File: core/translations/i18n/en.json [hunk 1]
```diff
@@ -49,82 +49,7 @@
   "Unloading module \"${0}\".": "Unloading module \"${0}\".",
   "What kind of shit platform is this that doesn't even support sending messages?": "What kind of shit platform is this that doesn't even support sending messages?",
   "Working on it...": "Working on it...",
-
```
Comment (ParasDPain) [inline core/translations/i18n/en.json]: > "Skipping \"${0}\". It isn't a Kassy module.": "Skipping \"${0}\". It isn't a Kassy module."

An instance of **Kassy** snuck in past the PR, across the other translation as well.

Comment (mrkno) [inline core/translations/i18n/en.json]: This is actually deliberate. Although the project is now called Concierge, the modules are still Kassy modules for convenience. (Its harder to spell `concierge.json` than `kassy.json` and I can't be bothered renaming every single one).

Comment (ParasDPain) [inline core/translations/i18n/en.json]: Oh! fair enough, though I ended up changing this to `Concierge module` in the translation PR; before I do another PR for that why don't we change `kassy.json` to `module.json` or `config.json` instead?

Comment (mrkno) [inline core/translations/i18n/en.json]: tbh I don't think I have the willpower to go through 40 repositories just to rename a file....

Comment (ParasDPain) [inline core/translations/i18n/en.json]: 48? Yeah let's keep it this way then 😆 ...as a memento to the original name as well

**Comment by phillipadsmith** [comment]
This PR indicates that is "Makes help optional," and yet I'm not seeing an obvious place to set that option...?

**Comment by mrkno** [comment]
@phillipadsmith Just don't provide a help function, it won't demand that you do

**Comment by phillipadsmith** [comment]
> @phillipadsmith Just don't provide a help function, it won't demand that you do

Oh, I see, I thought this was to make the built-in / default help optional, i.e., to replace the default with a customized response.

**Comment by mrkno** [comment]
Given help is now a module you could do that easily anyway
